sonar-db/src/main/java/org/sonar/db/purge/PurgeDao.java

```diff
@@ -178,18 +173,6 @@ private static void deleteProject(String rootUuid, PurgeMapper mapper, PurgeComm
     commands.deleteCeActivity(rootUuid);
   }
 
-  private void disableComponents(List<String> uuids, PurgeMapper mapper) {
-    executeLargeInputs(uuids,
-      input -> {
-        mapper.deleteResourceIndex(input);
-        mapper.setAnalysisIsLastToFalse(input);
-        mapper.deleteFileSourcesByUuid(input);
-        mapper.disableComponent(input);
-        mapper.resolveComponentIssuesNotAlreadyResolved(input, system2.now());
-        return emptyList();
-      });
-  }
-
   public void deleteAnalyses(DbSession session, PurgeProfiler profiler, List<IdUuidPair> analysisIdUuids) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUndocumentedApi)